deploy/helm/opa-operator/crds/crds.yaml

```diff
@@ -162,54 +162,44 @@ spec:
                                   description: The Microsoft Entra tenant ID.
                                   type: string
                                 tls:
-                                  default:
```

Comment:
I think it actually makes sense to default to webPki instead of null. I don't know what changes you did in Rust that resulted in this CRD change